ExampleHTTP/Podfile.lock

```diff
@@ -1,6 +1,6 @@
 PODS:
   - Keys (1.0.1)
-  - LibMobileCoin/CoreHTTP (6.0.0-pre1):
+  - LibMobileCoin/CoreHTTP (6.0.2):
```

Comment:
this needs a PR and release to cocoapods, or we can use its commit hash

Reply:
Hm, I don't think I manually changed this...

Reply:
It looks like this has been around for awhile - https://github.com/mobilecoinofficial/libmobilecoin/blob/master/LibMobileCoin.podspec (last changed 10 months ago).
I'm guessing it did get released?

Reply:
maybe it didn't get checked in and the submodule commit hash points to that version, so the lock file got updated.

Reply:
ya that must be the case. so its ok to leave as-is 6.0.2 is already released